### DISCUSSION TITLE
Adding additional CA Root certificate

### DISCUSSION THREAD

**Discussion by pva2007** [docusealco/docuseal#360] [Q&A]
Hi,

is there a way to add additional CA Root Certificate?

Background is the internal S3 using a local CA and when I try to use S3 I get the follwing error:
`E, [2024-09-11T11:05:01.881298 #1] ERROR -- : [c1d2120d-7eb7-415c-a41c-5793af074db0]
[c1d2120d-7eb7-415c-a41c-5793af074db0] Seahorse::Client::NetworkingError (SSL_connect returned=1 errno=0 peeraddr=192.168.xx.xx:443 state=error: certificate verify failed (unable to get local issuer certificate)):`

Thx upfront

**Comment by aHVzY2g** [comment]
You could mount `/etc/ssl/certs/ca-certificates.crt` from your host.
```
    volumes:
      - /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt:ro
```
